conbench/entities/history.py

```diff
@@ -143,7 +194,61 @@ def get_history(case_id: str, context_id: str, hardware_hash: str, repo: str) ->
         history_df, include_current_commit_in_rolling_stats=False
     )
 
-    return list(history_df.itertuples())
+    # return list(history_df.itertuples())
+
+    samples: list[HistorySample] = []
+    # Iterate over rows of pandas dataframe; get each row as namedtuple.
+    for sample in history_df.itertuples():
```

Comment:
What's non-obvious from the code above is the _names_ of the namedtuple attributes. 

They are largely derived from the attribute names in the `Session.query()` call above! In there, we for example see `BenchmarkResult.id`. The resulting dataframe then has a column with name `id`.